website/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "openritardi",
+    "version": "1.0.0",
+    "scripts": {
+        "build": "npx @11ty/eleventy"
+    },
+    "dependencies": {
```

Comment:
It should be "devDependencies" since the dependencies are only during development and not during production